### PR TITLE
8315606: Open source few swing text/html tests

### PR DESCRIPTION
**Pull request by prsadhuk**
Few closed swing text/html tests are opensourced

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8315606](https://bugs.openjdk.org/browse/JDK-8315606): Open source few swing text/html tests (**Bug** - P4)

### Reviewers
 * [Alexey Ivanov](https://openjdk.org/census#aivanov) (@aivanov-jdk - **Reviewer**) ⚠️ Review applies to [2b531b0b](https://git.openjdk.org/jdk/pull/15580/files/2b531b0b773e0a12acd45f21db25453fec65d3be)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/15580/head:pull/15580` \
`$ git checkout pull/15580`

Update a local copy of the PR: \
`$ git checkout pull/15580` \
`$ git pull https://git.openjdk.org/jdk.git pull/15580/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 15580`

View PR using the GUI difftool: \
`$ git pr show -t 15580`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/15580.diff">https://git.openjdk.org/jdk/pull/15580.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk/pull/15580#issuecomment-1707774071)

### CODE REVIEW

File: test/jdk/javax/swing/text/html/ImageView/bug4329185.java [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+ * questions.
+ */
+
+/*
+ * @test
+ * @bug 4329185
+ * @summary  Tests if vertical image alignment is working
+ * @key headful
+ * @run main bug4329185
+*/
```
Comment (aivanov-jdk) [inline test/jdk/javax/swing/text/html/ImageView/bug4329185.java:30]:
```suggestion
 * @run main bug4329185
 */
```

File: test/jdk/javax/swing/text/html/InlineView/bug4623342.java [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/* @test
+   @bug 4623342
+   @summary  Tests if InlineView causes extra spacing around images in JTable
+   @key headful
+   @run main bug4623342
+*/
```
Comment (aivanov-jdk) [inline test/jdk/javax/swing/text/html/InlineView/bug4623342.java:29]:
```suggestion
/*
 * @test
 * @bug 4623342
 * @summary  Tests if InlineView causes extra spacing around images in JTable
 * @key headful
 * @run main bug4623342
 */
```
To be consistent with other tests.

**Review by aivanov-jdk — APPROVED** [review]
Except for the minor formatting.

File: test/jdk/javax/swing/text/html/InlineView/bug4623342.java [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+        f.setVisible(true);
+    }
+
+    private void start() {
+        Shape r = jep.getBounds();
+        View v = jep.getUI().getRootView(jep);
+        int tableHeight = 0;
+        while (!(v instanceof javax.swing.text.html.ParagraphView)) {
+            int n = v.getViewCount();
+            Shape sh = v.getChildAllocation(n - 1,  r);
```
Comment (turbanoff) [inline test/jdk/javax/swing/text/html/InlineView/bug4623342.java:96]:
```suggestion
            Shape sh = v.getChildAllocation(n - 1, r);
```

**Pull request closed without merging**